install_config/router/default_haproxy_router.adoc

```diff
@@ -1122,6 +1122,44 @@ $ oc set env dc/router -c router DROP_SYN_DURING_RESTART=true
 If you used a non-default name for the router, you must change *_dc/router_*
 accordingly.
 
+[[deploy-router--arp-cach-tuning-for-large-scale-clusters]]
+== ARP Cache Tuning for Large-scale Clusters
+
+In {product-title}} clusters with large numbers of routes (greater than the
+value of `net.ipv4.neigh.default.gc_thresh3`, which is `1024` by default), you
+must increase the default values of sysctl variables to allow more entries in
+the ARP cache.
+
+In clusters, the kernel messages would be similar to the following:
+
+----
+[ 1738.811139] net_ratelimit: 1045 callbacks suppressed
+[ 1743.823136] net_ratelimit: 293 callbacks suppressed
+----
+
+When using the `oc` client, the commands fail and the user receives the following:
```

Comment:
When this issue occurs, you may begin to see the `oc` client commands being to fail with the following error: